cylc/flow/network/schema.py

```diff
@@ -1652,9 +1652,11 @@ class NamespaceName(String):
 
 
 class NamespaceIDGlob(String):
-    """A glob search for an active task or family.
+    """A task or family ID e.g. `2000/foo`.
 
-    Can use the wildcard character (`*`), e.g `foo*` might match `foot`.
+    Globs can be used to search for active tasks or families.
```

Comment:
I think "match" makes more sense in the context of doing a mutation, but no biggie
```suggestion
    Globs can be used to match active tasks or families.
```